airflow/models/dag.py

```diff
@@ -2297,7 +2297,7 @@ def dag_ready(dag_id: str, cond: BaseAsset, statuses: dict) -> bool | None:
         del all_records
         dag_statuses = {}
         for dag_id, records in by_dag.items():
-            dag_statuses[dag_id] = {x.asset.uri: True for x in records}
+            dag_statuses[dag_id] = {x.asset.name: True for x in records}
```

Comment:
Does this break things? If `name` is good enough, `uri` should be as well?

Reply:
This is used to `evaluate` and I think `uri` is probably good enough. 🤔 or should we just change it to (name, URI)?

Reply:
just revert it back. unless we decide to change it to (name, URI) next time